src/database/rulesConfig.ts

```diff
@@ -24,7 +24,7 @@ export function getRulesConfig(projectId: string, options: any): RulesInstanceCo
 
   if (!Array.isArray(dbConfig)) {
     if (dbConfig && dbConfig.rules) {
-      return [{ rules: dbConfig.rules, instance: options.instance }];
+      return [{ rules: dbConfig.rules, instance: options.instance || options.project }];
```

Comment:
This is the bug.  We now fall back to the project Id when there is no `instance` in `options`.